core/src/main/java/org/dozer/DozerBeanMapperBuilder.java

```diff
@@ -264,6 +270,36 @@ public DozerBeanMapperBuilder withSettingsProcessor(SettingsProcessor processor)
     }
 
     /**
+     * Registers a {@link ELEngine} for the mapper.
+     * Which can be used to resolve expressions within the defined mappings.
+     * <p>
+     * By default, {@link NoopELEngine} is registered.
+     *
+     * @param elEngine elEngine to use
+     * @return modified builder to be further configured.
+     */
+    public DozerBeanMapperBuilder withELEngine(ELEngine elEngine) {
+        this.elEngine = elEngine;
+        return this;
+    }
+
+    /**
+     * Registers a {@link ELEngine} and {@link ElementReader} for the mapper.
+     * Which can be used to resolve expressions within the defined XML mappings.
+     * <p>
+     * By default, {@link NoopELEngine} and {@link SimpleElementReader} are registered.
+     *
+     * @param elEngine elEngine to use
+     * @param elementReader elementReader to use
+     * @return modified builder to be further configured.
+     */
+    public DozerBeanMapperBuilder withELEngineAndElementReader(ELEngine elEngine, ElementReader elementReader) {
```

Comment:
Just some thoughts here.
I would keep these two configurations separately as they do different things. 
And I would say that with introduction of `NoopELEngine` we do not need to have `ExpressionElementReader` and `SimpleElementReader` any more - `ExpressionElementReader`  seems to be safe to use always, as with  `NoopELEngine` it will do exactly what `SimpleElementReader` does, and ELEngine is a mandatory component now.

Reply:
Updated as requested.